lib/test/karma.js

```diff
@@ -38,29 +43,35 @@ function getKarmaReporterPluginPath(reporterName) {
     throw 'Get Karma Reporter Plugin Path: Reporter name is unspecified';
   }
 
-  if (reporterName === defaultReporterName) {
-    return defaultReporterName;
+  //Default reporter is a dependency of angularity
+  var reporterPath = (reporterName === defaultReporterName) ?
+    path.resolve(__dirname, '../../node_modules', defaultReporterName) :
+    reporterName;
+  //Specific reporters are expected to be dependencies of the local project
+  reporterPath = (path.dirname(reporterPath) === '.') ?
+    path.resolve('node_modules', 'karma-' + reporterName + '-reporter') :
```

Comment:
What is the best convention, to specify `karma-foo-reporter` or `foo` ?
@impaler What is your oppinion?